src/WebManager/app/src/components/manifest-viewer.tsx

```diff
@@ -101,26 +100,36 @@ export class ManifestViewer extends React.Component<IManifestViewerProps, IManif
                     props.push({ key: key, value: this.renderValue(value[key]) });
                 }
             }
+
         }
 
         let el = (
-            <div className="ms-Grid ms-font-m">
-                {props.map(x => (
-                    <div className="ms-Grid-row manifest-entry" key={x.key}>
-                        <div className="ms-Grid-col ms-u-sm2">
-                            <span>{x.key}</span>
-                        </div>
-                        <div className="ms-Grid-col ms-u-sm10">
-                            {x.value}
+
+            <div>
+                <div className="ms-Grid ms-font-m">
+                    {props.map(x => (
+                        <div className="ms-Grid-row manifest-entry" key={x.key}>
+                            <div className="ms-Grid-col ms-u-sm2">
+                                <span>{x.key}</span>
+                            </div>
+                            <div className="ms-Grid-col ms-u-sm10">
+                                {x.value}
+                            </div>
                         </div>
-                    </div>
-                ))}
+                    ))}
+                </div>
+
             </div>
+
+
+
+
```

Comment:
There's a large number of extra lines in this file (here, and both above and below). This is not technically an issue, but it makes for a lot of empty real estate on the screen when working with the file. Think of empty space strategically, as it conveys meaning in usability and readability of the code.